assets/i18n/qqq.json

```diff
@@ -138,6 +138,7 @@
 	"I18N_DELETE_ACCOUNT_PAGE_PARAGRAPH_3": "Text on the Delete Account page that explains the details of user account deletion.",
 	"I18N_DELETE_ACCOUNT_PAGE_PARAGRAPH_4": "Text on the Delete Account page that explains the details of user account deletion.",
 	"I18N_DELETE_ACCOUNT_PAGE_PARAGRAPH_5": "Text on the Delete Account page that explains the details of user account deletion.",
+	"I18N_DELETE_ACCOUNT_PAGE_REDIRECT_INFO": "Label for the Delete Account page that allows the user to delete their account.",
```

Comment:
The value here should explain the usage of the text.

Maybe: `Text that explain to the user that clicking the button will redirect to a new page.` (Or something around that?)

Reply:
Made the change to suit the label. Should be fixed now!